modules/mineapi/src/commands/tps.ts

```diff
@@ -0,0 +1,110 @@
+// Import Types
+import { Command } from "@spookelton/wrapperHelpers/types";
+import type { MessageEmbedOptions } from "discord.js";
+
+import { thread, getCore } from "..";
+import { helpHelper } from "@spookelton/wrapperHelpers/modul";
+import { hex } from "@spookelton/wrapperHelpers/colors"
+import { ColorKey } from "@spookelton/wrapperHelpers/types"
+
+type TpsResponse = {
+	dimensions: {
+		dimId: string,
+		dimName: string | null,
+		tickTime: number,
+		tps: number,
+	}[],
+	overall: {
+		tickTime: number,
+		tps: number,
+	},
+}
+
+function parseOutput(input: RegExpMatchArray[]): TpsResponse {
+	// We don't want the Overall tps to be parsed as a dimension, so remove it and store it for later
+	const inputOverall = input.pop();
+	if (!inputOverall) throw new Error("It seems like something bad happened I'll change this error later");
+
+	const output = {
+		dimensions: input.map(x => ({
+			dimId: x[1].replace(/Dim  ?/, ""),  // Remove extra "Dim" at the start
+			dimName: x[2] === "null" ? null : x[2],
+			tickTime: parseFloat(x[3]),
```

Comment:
You should use ```Number()``` or ```+``` to convert strings to number instead of ```parseFloat``` unless you know you need ```parseFloat```.

Ideally use ```+```.

Ex:
```ts
tickTime: +x[3];
```

Reply:
Yeah figured that out, gotta get used to typescript